huvr_client/client.py

```diff
@@ -68,6 +68,7 @@ def __init__(
 
         self.verbose = verbose
         self.session = requests.Session()
+        self.session.headers["User-Agent"] = "huvr_client/0.3.7 python"
```

Comment:
I missed this, but we should not have hardcoded the release number
I wrote a new issue for this: https://github.com/huvrdata/huvr-client/issues/30